app/models/search-result.ts

```diff
@@ -211,7 +211,8 @@ export default class SearchResultModel extends Model {
         if (this.resourceMetadata.isContainedBy?.[0]?.rights) {
             return {
                 name: this.resourceMetadata.isContainedBy[0].rights?.[0]?.name?.[0]?.['@value'],
```

Comment:
And I looked up a line and saw that it also has a bunch of `.[0]`s. Man, who CR'd this thing before?